src/batou_ext/geoip.py

```diff
@@ -24,6 +24,7 @@ class GeoIPDatabase(batou.component.Component):
     def configure(self):
 
         self.provide('geoip_database', self)
+        self.download_url = self.expand(self.download_url)
```

Comment:
this is fixed in batou 2.3b3 where you can say:

Attribute(str, default_conf_string="https://.....{{component.license_key}}")